### PR TITLE
Style change

### PR DESCRIPTION
**Pull request by jeffheifetz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 language: node_js
 node_js:
   - 'stable'
-  - '0.12'
-  - '0.10'
+  - '6.0.0'
```
Comment (turbobeast) [inline .travis.yml:4]: how did the old test pass, if the node version was `0.12` and `0.10`? there doesn't seem to be a compile step in the test command.

File: test/index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+describe('electron/oauth2', function () {
+  it('should not throw an error', function () {
+    require('../index');
```
Comment (turbobeast) [inline test/index.js:5]: you don't want to add a `coverage` badge?

File: .travis.yml (hunk 1 above)
Comment (jeffheifetz) [inline .travis.yml:4]: Correct, the only tests running previously was XO a linter

**Comment by turbobeast** [comment]
if you squash it, I'll merge it

**Comment by jeffheifetz** [comment]
This just got merged to upstream master unsquased (https://github.com/mawie81/electron-oauth2/pull/9) so I'll just close for now

**Pull request closed without merging**